receiver/pluginreceiver/README.md

```diff
@@ -0,0 +1,83 @@
+# Plugin Receiver
+
+The Plugin Receiver is designed to run templated OpenTelemetry pipelines. This allows users to store complex workflows within a plugin that is loaded by the receiver.
+
+Supported pipeline types: `logs`, `metrics`, `traces`
+
+## Configuration
+| Field        | Default | Required | Description |
+| ---          | ---     | ---      | ---         |
+| `path`       |         | `true`   | The path to the plugin file. |
+| `parameters` | { }     | `false`  | A map of `key: value` parameters used to render the plugin's templated pipeline. |
+
+### Example Configuration
+```yaml
+receivers:
+  plugin:
+    path: ./plugins/simplehost.yaml
+    parameters:
+      enable_cpu: false
+      enable_memory: true   
+```
+
+## Plugins
+Plugins are yaml files that define three key aspects:
+- Metadata
+- Parameters
+- Template
+
+### Example
+```yaml
+title: Simple Host Plugin
+description: A plugin utilizing the hostmetrics receiver
+version: 0.0.0
+parameters:
+- name: enable_cpu
+  type: bool
+  default: false
+- name: enable_memory
+  type: bool
+  default: true
+template: |
+  receivers:
+    hostmetrics:
+      scrapers:
+  {{if .enable_cpu}}
+        cpu:
+  {{end}}
+  {{if .enable_memory}}
+        memory:
+  {{end}}
+  service:
+    pipelines:
+      metrics:
+        receivers: [hostmetrics]
+```
+### Metadata
+Metadata fields are used to catalog and distinguish plugins. The following fields are used for metadata:
```

Comment:
Are all these fields required?

Reply:
They are not required at the moment. Do we want to enforce this with a validation step after we unmarshal to the plugin struct?

Reply:
I don't think we need to but I think it's worth calling out if they are required or not. I know if I'm setting one of these up to test it would be nice to know the minimum set of things I need in order to get going.

Reply:
Okay. I think that's a fair compromise. I'll update the documentation.